frontend/src/service/httpService.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import store from '../store';
+import authCookie from './auth';
+import configuration from '../config';
+import state from '../state';
+
+const httpService = axios.create({
```

Comment:
You created a little incorrectly, I think service has to contain two methods request() and authRequest(), in order to you can use it like this

```
httpService.authRequest({
                url: configuration.LogoutUri,
                method: 'post',
                data: {
                    user
                }
            }).
```

So it is well ` axios.create({`, but store it in another variable and use it inside methods